vms/platformvm/validators/manager.go

```diff
@@ -283,19 +292,22 @@ func (m *manager) makePrimaryNetworkValidatorSet(
 		validatorSet,
 		currentHeight,
 		lastDiffHeight,
-		constants.PrimaryNetworkID,
+		subnetID,
 	)
 	return validatorSet, currentHeight, err
 }
 
-func (m *manager) getCurrentPrimaryValidatorSet(
+func (m *manager) getCurrentValidatorSet(
 	ctx context.Context,
+	subnetID ids.ID,
 ) (map[ids.NodeID]*validators.GetValidatorOutput, uint64, error) {
-	primaryMap := m.cfg.Validators.GetMap(constants.PrimaryNetworkID)
+	subnetMap := m.cfg.Validators.GetMap(subnetID)
 	currentHeight, err := m.getCurrentHeight(ctx)
-	return primaryMap, currentHeight, err
+	return subnetMap, currentHeight, err
 }
 
+// TODO: Once Etna has been activated, remove this function and use
+// makeValidatorSet for all validator set lookups.
```

Comment:
The TODO on line 365 is no longer relevant since now after Etna `makeValidatorSet` should be used instead of `makeSubnetValidatorSet`, right?

Reply:
🦅 👁️